Formula/offlineimap.rb

```diff
@@ -24,15 +24,18 @@ def caveats; <<-EOS.undent
     EOS
   end
 
-  test do
-    system bin/"offlineimap", "--version"
-  end
+  plist_options :startup => true
```

Comment:
What does manual actually mean? I figured startup means that the service should be started upon login if configured by the user and manual describes some manual method the user can use to manually start the service. What command should go into the manual part just the same as the one in the plist without the full path?